page.css

```diff
@@ -1,5 +1,4 @@
 ---
-layout: null
```

Comment:
Yeah, that is the whole point.
On May 15, 2015 11:11 PM, "░▒▓█│【Walkman】│█▓▒░" <notifications@github.com>
wrote:

> In page.css
> <https://github.com/mdo/github-buttons/pull/79#discussion_r30441847>:
>
> > @@ -1,5 +1,4 @@
> >  ---
> > -layout: null
>
> I meant is the css file still rendered with jekyll...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdo/github-buttons/pull/79/files#r30441847>.
>